integration-tests/tests/src/test/java/biz/netcentric/vlt/upgrade/groovy/GroovyInstallFailedIT.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * (C) Copyright 2017 Netcentric AG.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package biz.netcentric.vlt.upgrade.groovy;
+
+import org.apache.sling.testing.clients.ClientException;
+import org.junit.Before;
+import org.junit.Test;
+
+public class GroovyInstallFailedIT extends GroovyAbstractIT {
+
+    private String testPropertyValue;
+
+    @Before
+    public void setUp() throws Exception {
+        super.setUp();
+        testPropertyValue = System.getProperty("vaultUpgradeHook.testpackage.groovy.installfailed.testPropertyValue");
+    }
+
+    @Test
+    public void shouldExecuteFailedScript() throws ClientException, InterruptedException {
+        assertTestPropertyValue(testResourcePath, testPropertyName, testPropertyValue);
```

Comment:
@nc-andreashaller, when the package contains some conflicting changes (like nt:unstructured under nf:folder) so they can not be saved - then out status cannot be saved as well (and I can not assert it).

Is it an expected behavior?

Reply:
opened a separate issue for this #37